lib/ex_webrtc/sdp_utils.ex

```diff
@@ -46,6 +46,46 @@ defmodule ExWebRTC.SDPUtils do
     |> ExSDP.Media.add_attributes(attributes ++ media_formats)
   end
 
+  # TODO: refactor
```

Comment:
I would get rid of TODO: refactor. Let's write a code that is acceptable to a degree that it can be merged and I believe this one can be merged :stuck_out_tongue: